contracts/test/unit/pcv/utils/ERC20Skimmer.t.sol

```diff
@@ -0,0 +1,167 @@
+pragma solidity =0.8.13;
+
+import {Vm} from "./../../utils/Vm.sol";
+import {ICore} from "../../../../core/ICore.sol";
+import {DSTest} from "./../../utils/DSTest.sol";
+import {MockERC20} from "../../../../mock/MockERC20.sol";
+import {ERC20Skimmer} from "../../../../pcv/utils/ERC20Skimmer.sol";
+import {ERC20HoldingPCVDeposit} from "../../../../pcv/ERC20HoldingPCVDeposit.sol";
+import {getCore, getAddresses, VoltTestAddresses} from "./../../utils/Fixtures.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "hardhat/console.sol";
+
+contract UnitTestERC20Skimmer is DSTest {
+    ICore private core;
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+    VoltTestAddresses public addresses = getAddresses();
+
+    /// @notice reference to the PCVDeposit to pull from
+    ERC20HoldingPCVDeposit private pcvDeposit0;
+
+    /// @notice reference to the PCVDeposit to pull from
+    ERC20HoldingPCVDeposit private pcvDeposit1;
+
+    /// @notice reference to the PCVDeposit that cannot be pulled from
+    ERC20HoldingPCVDeposit private pcvDeposit2;
+
+    /// @notice reference to the ERC20
+    ERC20Skimmer private skimmer;
+
+    /// @notice token to push
+    MockERC20 private token;
+
+    /// @notice threshold over which to pull tokens from pull deposit
+    uint248 private constant targetBalance = 100_000e18;
+
+    /// @notice maximum rate limit per second in RateLimitedV2
+    uint256 private constant maxRateLimitPerSecond = 1_000_000e18;
+
+    /// @notice rate limit per second in RateLimitedV2
+    uint128 private constant rateLimitPerSecond = 10_000e18;
+
+    /// @notice buffer cap in RateLimitedV2
+    uint128 private constant bufferCap = 10_000_000e18;
+
+    /// @notice target address where funds will be sent
+    address target = address(100_000_000);
+
+    function setUp() public {
```

Comment:
add integration test for this skimmer where it sends comp to multisig or mock PSM

Reply:
done